deploy.ci.functional.mariadb.prefer-lowest.dynamic-store.yml

```
@@ -6,7 +6,7 @@ tag: '1.0'
 environment: docker.ci
 pipeline: docker.ci.functional.dynamic-store
 image:
-    tag: spryker/php:8.1
+    tag: spryker/php:8.2
```

Prefer-lowest jobs should stay 8.1
```suggestion
    tag: spryker/php:8.1
```